### PR TITLE
chore: set explicit 2024.3 version and enable qs in the project view

### PR DESCRIPTION
**Pull request by tpasternak**
# Checklist

- [ ] I have filed an issue about this change and discussed potential changes with the maintainers.
- [ ] I have received the approval from the maintainers to make this change.
- [ ] This is not a stylistic, refactoring, or cleanup change.

Please note that the maintainers will not be reviewing this change until all checkboxes are ticked. See 
the [Contributions](https://github.com/bazelbuild/intellij#contributions) section in the README for more 
details.

# Discussion thread for this change

Issue number: `<please reference the issue number or url here>`

# Description of this change

### CODE REVIEW

File: ijwb/ijwb.bazelproject [hunk 1]
```diff
@@ -14,11 +14,13 @@ targets:
 workspace_type: intellij_plugin
 
 build_flags:
-  --define=ij_product=intellij-latest
+  --define=ij_product=intellij-2024.3
```
Comment (LeFrosch) [inline ijwb/ijwb.bazelproject:17]: Maybe intllij-oss-under-dev?

**Review by LeFrosch — CHANGES_REQUESTED** [review]

File: ijwb/ijwb.bazelproject (hunk 1 above)
Comment (LeFrosch) [reply ijwb/ijwb.bazelproject:17]: Because, sooner or later this will be outdated.

**Comment by LeFrosch** [comment]
Also I don't think these `ijwb` and `clwb` specific configurations make sense with qsync. Since, the main benefit of qsync is that you can just import what you need. Maybe we can just add on `.bazelproject` file for the repo?

File: ijwb/ijwb.bazelproject (hunk 1 above)
Comment (dkashyn-sfdc) [reply ijwb/ijwb.bazelproject:17]: `intellij-latest` is outdated by itself and using some dated version which is far from being "latest"

**Comment by tpasternak** [comment]
> Maybe we can just add on .bazelproject file for the repo
Yeah, you're right!

File: ijwb/ijwb.bazelproject (hunk 1 above)
Comment (LeFrosch) [reply ijwb/ijwb.bazelproject:17]: Sure, I don't mind the change. If you think it makes it easier for external contributors go ahead.

**Review by LeFrosch — APPROVED** [review]

**Pull request merged**